test/e2e/broker_event_transformation_test.go

```diff
@@ -0,0 +1,159 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	"github.com/knative/eventing/test"
+	pkgTest "github.com/knative/pkg/test"
+	"k8s.io/apimachinery/pkg/util/uuid"
+)
+
+/*
+TestEventTransformationForTrigger tests the following scenario:
+
+                         5                 4
+                   ------------- ----------------------
+                   |           | |                    |
+             1     v	 2     | v        3           |
+EventSource ---> Broker ---> Trigger1 -------> Service(Transformation)
+                   |
+                   | 6                   7
+                   |-------> Trigger2 -------> Service(Logger)
+
+Note: the number denotes the sequence of the event that flows in this test case.
+*/
+
```

Comment:
Remove the empty line between the comment and the function.

Reply:
Done.